redfish-core/lib/pcie.hpp

```diff
@@ -500,6 +500,88 @@ inline void
     });
 }
 
+inline void
+    getPCIeDevicePrettyName(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                            const std::string& pcieDevicePath,
+                            const std::string& service)
+{
+    sdbusplus::asio::getAllProperties(
+        *crow::connections::systemBus, service, pcieDevicePath,
+        "xyz.openbmc_project.Inventory.Item",
+        [pcieDevicePath, asyncResp{asyncResp}](
```

Comment:
Is it sufficient to use `[asyncResp]` (instead of `[asyncResp{asyncResp}]`)?

Reply:
yes, done